app/src/main/res/layout/activity_main.xml

```diff
@@ -30,6 +30,5 @@
             app:layout_constraintStart_toStartOf="parent"
             app:menu="@menu/bottom_navigation_menu" />
 
```

Comment:
i would completely drop any empty line here, as we have closing elements here

Reply:
Ok, I see it but I think this style with empty separate line make it more readable and I think it is not incorrect. I believe I had the same style in every XML file.  Do you @marcinWisnia  and @BorysChajdas think I should in every layout file delete the empty line?

Reply:
I created an issue for this #163.